### PR TITLE
implement loguru

### PR DESCRIPTION
**Pull request by Alliad**

### CODE REVIEW

File: mantis/blackbox/cvm_indexer.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from typing import List, Optional
 
+from custom_logging import logger
```
Comment (dzmitry-lahoda) [inline mantis/blackbox/cvm_indexer.py:5]: will not just using loguru directly work? not abstraction needed.

Comment (Alliad) [reply mantis/blackbox/cvm_indexer.py:5]: This abstruction for extracting logs from uvicorn.
But we can use directly work if so

Comment (Alliad) [reply mantis/blackbox/cvm_indexer.py]: Switched to use original loguru fuctionality. But for common logging config need to implement 
https://github.com/erezinman/loguru-config

Comment (dzmitry-lahoda) [reply mantis/blackbox/cvm_indexer.py]: yeah, do not delete it, thank you. may be will be needed at some. when will get heavy load on server.

**Pull request merged**